client/src/components/Jobs/Jobs.scss

```diff
@@ -89,8 +89,3 @@
     /* date */    tr > *:nth-child(6) { width: 14%; }
   }
 }
-
-/* Always */
-.job__status {
```

Comment:
the use of `job__status` should be dropped in `Jobs.js`

Reply:
Thanks for catching that, Nathan!  I've removed the `job__status` className from `Jobs.js`.